src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -1098,7 +1098,27 @@ var _ = Describe("installer HostRoleMaster role", func() {
 				assistedController.UpdateNodeLabels(context.TODO(), &wg)
 				wg.Wait()
 			})
+			It("Set label only in Ready node", func() {
+				nodeLabels := `{"node.ocs.openshift.io/storage":""}`
+				k8sNodes := GetKubeNodes(kubeNamesIds)
```

Comment:
```suggestion
				notReadyk8sNodes := GetKubeNodes(kubeNamesIds)
```

Reply:
not right, we have only 1 not ready node and i pretty ok with calling it k8sNodes